CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required( VERSION 2.8 FATAL_ERROR )
 project (urdfdom_headers)
 
+include(GNUInstallDirs)
```

Comment:
We seem to be making an effort to keep urdfdom fairly platform independent these days (for example, we have accepted changes for supporting Win32 and Cygwin). Is GNUInstallDirs a Linux (GNU, even) specific thing? ping @scpeters for his opinion on cross-platform compatibility.